### PR TITLE
Fix #48460 stuttgarter-zeitung.de

### LINKED ISSUE #48460: stuttgarter-zeitung.de
### Issue URL (Annoyance)
[https://www.stuttgarter-zeitung.de/inhalt.banksy-in-der-staatsgalerie-schredderbild-verlaesst-stuttgart.3664cf54-3eba-4c9b-a482-ed46a3f193b6.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.stuttgarter-zeitung.de%2Finhalt.banksy-in-der-staatsgalerie-schredderbild-verlaesst-stuttgart.3664cf54-3eba-4c9b-a482-ed46a3f193b6.html)
### Comment
Username: @Morku
> When wait for a while, a Pop-up appear, telling you the homepage get refreshed and offer to reload, interrupting you reading the article. It's sporadic then.
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (180)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard German

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#48460

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@BlazDT please take a look

File: GermanFilter/sections/adservers.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ||yieldlove-ad-serving.net^
+||powerstories.de^$third-party
```
Comment (BlazDT) [inline GermanFilter/sections/adservers.txt]: Not requested on my device.

File: GermanFilter/sections/specific.txt [hunk 2]
```diff
@@ -67,6 +67,7 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
 !
 carsharing-news.de##.inside-right-sidebar > aside#custom_html-2 > .textwidget > a[href][target="_blank"] > img
 coolis.de##.td-ss-main-sidebar > aside.widget_text > .textwidget > .g-1 > .g-single > a.gofollow[href] > img
+stuttgarter-zeitung.de##.mod-teaser--sondt
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt]: Also not shown.

**Review by BlazDT — APPROVED** [review]

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply GermanFilter/sections/specific.txt]: <details>
<summary>Here it is</summary>

</details>

**Review by zloyden — APPROVED** [review]

**Pull request merged**